eframe/src/native/epi_integration.rs

```diff
@@ -8,6 +8,25 @@ pub fn points_to_size(points: egui::Vec2) -> winit::dpi::LogicalSize<f64> {
     }
 }
 
+pub fn read_window_info(window: &winit::window::Window) -> Option<WindowInfo> {
+    match window.outer_position() {
+        Ok(pos) => {
+            let size = window.inner_size();
+            Some(WindowInfo {
+                position: egui::Vec2 {
+                    x: pos.x as f32,
+                    y: pos.y as f32,
+                },
+                size: egui::Vec2 {
+                    x: size.width as f32,
+                    y: size.height as f32,
```

Comment:
`outer_position()` returns physical pixels, but egui uses logical pixels (called "points") almost everywhere. Most importantly, eframe uses points for `Frame::set_window_size` and `Frame::set_window_pos`, so we should be consistent with that methinks.

You can use `window.outer_position().to_logical::<f32>(egui_ctx.pixels_per_point())` to get the size in logical points (egui coordinates).

Reply:
thanks for catching that, fixed.